src/libs/KeyboardShortcut/bindHandlerToKeydownEvent/index.native.ts

```diff
@@ -0,0 +1,28 @@
+import getKeyEventModifiers from '../getKeyEventModifiers';
+import BindHandlerToKeydownEvent from './types';
+
+/**
+ * Checks if an event for that key is configured and if so, runs it.
+ */
+const bindHandlerToKeydownEvent: BindHandlerToKeydownEvent = (getDisplayName, eventHandlers, keyCommandEvent, event) => {
+    const eventModifiers = getKeyEventModifiers(keyCommandEvent);
+    const displayName = getDisplayName(keyCommandEvent.input, eventModifiers);
+
+    // Loop over all the callbacks
+    Object.values(eventHandlers[displayName]).every((callback) => {
```

Comment:
Regression from this PR. Sometimes I get crash:

<img width="332" alt="crash" src="https://github.com/Expensify/App/assets/108292595/64e00543-fe58-4791-94d2-1b56bf880026">

Reply:
Any specific flow/screen you're seeing this?

Reply:
Happens on all platforms

Reply:
@blazejkustra can you please raise quick PR? This is now deployed to staging

Reply:
On it 👍

Reply:
@blazejkustra this is already fixed i think https://github.com/Expensify/App/pull/30602

Reply:
Yep, I just noticed after merging the newest main, thanks!